src/blocktypes/proviso/publish.sh

```diff
@@ -0,0 +1,29 @@
+SKOPEO=${SKOPEO:-$(dirname "${BASH_SOURCE[0]}")/skopeo.sh}
+
+function proviso() {
+  local -n input=$1
+  local -n output=$2
+
+  local missing tag
+  local -a images filtered
+
+  missing=$(mktemp)
+  trap 'rm -f $missing' RETURN
+
+  mapfile -t images < <(jq -r '.meta.tags[0]|select(.!=null)' <<<"${input[@]}")
+
+  echo "${images[@]}" |
+    command xargs -n 1 -P 0 "$SKOPEO" "$missing"
```

Comment:
By `$SKOPEO` name I was enticed to beleive this is what skopeo normally does in this context, but no, it's more like `$SKOPEO_INSPECT`. Reading it this way, this line starts to make more sense.

Can we rename?

Reply:
sure